client/src/GlobalStyles.js

```diff
@@ -37,6 +37,27 @@ function GlobalStyles() {
           font-size: 14px;
           color: var(--font-color-main);
           background-color: var(--bg-color-main);
+          button:active,
+          button:focus,
+          input:focus,
+          input:active {
+            outline: none;
+          }
+
+          button,
+          a {
+            cursor: pointer;
+          }
+
+          h2,
+          h3,
+          h4,
+          h5 {
+            margin: 0;
+          }
```

Comment:
I suggest putting in a break in this line for a better readability and consistency <3